<html>
<body>
Обнаружен вызов метода <code>java.lang.Class.newInstance()</code>.
<p>
  Этот метод распространяет исключения, выбрасываемые конструктором без аргументов, включая проверяемые исключения. Использование этого метода позволяет эффективно обойти проверку исключений времени компиляции, выполняемую компилятором.
</p>
<p>
  Рекомендуется использовать быстрое исправление для замены этого вызова на вызов метода <code>java.lang.reflect.Constructor.newInstance()</code>, который обрабатывает любые исключения, выбрасываемые конструктором, обертывая их в (проверяемый) <code>java.lang.reflect.InvocationTargetException</code>, что позволяет избежать данной проблемы.
</p>
<p><b>Пример:</b></p>
<pre><code>
  clazz.newInstance()
</code></pre>
<p>После применения быстрого исправления:</p>
<pre><code>
  clazz.getConstructor().newInstance();
</code></pre>
</body>
</html>
